articles/quickstart/webapp/symfony/01-login.md

```diff
@@ -193,18 +186,59 @@ Notice that we need to identify the user provided selected in the step before bo
 
 ## Trigger Authentication
 
-Set the following in `app/resources/views/index.html.twig`
+Set the following in `app/resources/views/default/index.html.twig`
 
 ```html
 {% if app.user %}
     Welcome, {{ app.user.username }}!<br/>
     {{ dump(app.user) }}
     <a href="{{ url('secured') }}">Protected route</a>
-    <a href="{{ logout_url("secured_area") }}">
+    <a href="{{ logoutUrl }}">
         <button>Logout</button>
     </a>
 {% else %}
     <h1>Symfony Auth0 Quickstart</h1>
     <a href="/connect/auth0"><button>Login</button></a>
 {% endif %}
 ```
+
+## Logout
+
+To log the user out, you have to clear the data from the session, and redirect the user to the Auth0 logout endpoint. You can find more information about this in the [logout documentation](/logout).
+
+Add this to your `src/AppBundle/Controller/DefaultController.php`.
+
+```php
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function indexAction()
+    {
+        $returnTo = sprintf('%s://%s:%s/auth0/logout',
```

Comment:
We probably don't want to include the colon here if there is no port number (which is the majority of cases that aren't localhost). I'm just worried that someone will copy this into their code and wonder why it's not working.

Reply:
Also ... I assume that `/auth0/logout` is where the application logout happens, right? That path is set somewhere else in the seed project?

Reply:
You're right, `/auth0/logout` is where the application logout happens. That path is set in `routing.yml` and `security.yml`.